src/category_theory/monad/monadicity.lean

```diff
@@ -62,7 +62,7 @@ The "main pair" for an algebra `(A, α)` is the pair of morphisms `(F α, ε_FA)
 reflexive pair, and will be used to construct the left adjoint to the comparison functor and show it
 is an equivalence.
 -/
-instance main_pair_reflexive (A : algebra (F ⋙ G)) :
+instance main_pair_reflexive (A : adj .to_monad.algebra) :
```

Comment:
Is there some reason for the extra space here in `adj .to_monad`? It looks funny/unnecessary.

Reply:
Yeah, it's because `adj` here is a notation made above, and it doesn't recognise the notation if there's no space.

Reply:
The alternatives are `(adj).to_monad.algebra` (which I think looks just as weird) or `adjunction.to_monad.algebra adj` which doesn't look too bad but is quite wordy

Reply:
I see. In any case this is orthogonal to this PR -- I'd missed these when they entered mathlib. I think a perfectly fine fix would be to add a comment (either at the point the local notation is defined, or the first place where a mysterious space appears, or at both places) explaining that the space is necessary. Whether or not it happens in this PR, I think this is confusing and the reader deserves help. :-)

Reply:
There was some discussion on this here: https://github.com/leanprover-community/mathlib/pull/5137#discussion_r541519242 - I think that after this PR, there's scope to generalise some monad statements and avoid some of the weird spaces, but I'll amend for now :)